website/docs/provisioners/habitat.html.markdown

```diff
@@ -47,20 +47,25 @@ There are 2 configuration levels, `supervisor` and `service`.  Configuration pla
 
 ### Supervisor Arguments
 * `version (string)` - (Optional) The Habitat version to install on the remote machine.  If not specified, the latest available version is used.
-* `use_sudo (bool)` - (Optional) Use `sudo` when executing remote commands.  Required when the user specified in the `connection` block is not `root`.  (Defaults to `true`)
+* `license (string)` - (Optional) Sets the acceptance of Chef licensing: https://www.chef.io/end-user-license-agreement/
+* `auto_update (bool)` - (Optional) If set to `true`, the supervisor will auto-update itself as soon as new releases are available on the specified `channel`.
+* `http_disable (bool)` - (Optional) If set to `true`, disables the supervisor HTTP listener entirely.
+* `peers (array)` - (Optional) A list of IP or FQDN's of other supervisor instance(s) to peer with. (Defaults to none)
 * `service_type (string)` - (Optional) Method used to run the Habitat supervisor.  Valid options are `unmanaged` and `systemd`.  (Defaults to `systemd`)
 * `service_name (string)` - (Optional) The name of the Habitat supervisor service, if using an init system such as `systemd`. (Defaults to `hab-supervisor`)
-* `peer (string)` - (Optional) IP or FQDN of a supervisor instance to peer with. (Defaults to none)
+* `use_sudo (bool)` - (Optional) Use `sudo` when executing remote commands.  Required when the user specified in the `connection` block is not `root`.  (Defaults to `true`)
```

Comment:
default to `true` on `sudo`?

Reply:
I don't recall changing that option, I think maybe I re-ordered it a bit?  The original default was `true` from previous commits (https://github.com/hashicorp/terraform/blob/master/website/docs/provisioners/habitat.html.markdown#supervisor-arguments), and matches the Provisoner's schema default value (https://github.com/hashicorp/terraform/blob/master/builtin/provisioners/habitat/resource_provisioner.go#L89).

Did you want me to change the default to `none` in both locations?

Reply:
Ah foiled by diff reading! No, leave as-is 👍